tests/ScopeTest.php

```diff
@@ -298,7 +298,7 @@ public function testConditionOnReferencedRecords()
             $user->addCondition('Tickets/user/country_id/Users/country_id/Users/name', '!=', null); // should be always true
         }
 
-        $this->assertEquals(2, $user->toQuery('count')->getOne());
+        $this->assertEquals(2, $user->toQuery()->count()->getOne());
```

Comment:
This may look better at the first sight, but:
- can `$user->toQuery()` be reused? I think no.
- what is the purpose of "uninitialized/unbuilded" query object (`$user->toQuery()`)?

Reply:
Purpose is to access the raw query / data.
In case of `count` I would say we can introduce a method `Model::count` directly

Reply:
I highly prefer to not enlarge Model, we should to exactly the oposite